plugin.video.vrt.nu/resources/lib/kodiwrappers/kodiwrapper.py

```diff
@@ -40,8 +40,16 @@ def __init__(self, handle, url, addon):
 
     def show_listing(self, list_items, sort=None, content_type='episodes'):
         listing = []
+
+        if sort is not None:
+            kodi_sorts = {sortmethod.ALPHABET: xbmcplugin.SORT_METHOD_LABEL_IGNORE_THE}
+            kodi_sortmethod = kodi_sorts.get(sort)
+            xbmcplugin.addSortMethod(handle=self._handle, sortMethod=kodi_sortmethod)
+        else:
+            xbmcplugin.addSortMethod(handle=self._handle, sortMethod=xbmcplugin.SORT_METHOD_NONE)
+
         for title_item in list_items:
-            list_item = xbmcgui.ListItem(label=title_item.title)
+            list_item = xbmcgui.ListItem(label=title_item.title, thumbnailImage=title_item.art_dict.get('thumb'))
```

Comment:
I thought that `thumbnailImage` was deprecated ?

Reply:
You're right. It can be removed again.